ui/components/component-library/button-secondary/button-secondary.scss

```
@@ -2,7 +2,9 @@
   &:hover:not(&--disabled) {
     color: var(--color-primary-inverse);
     background-color: var(--color-primary-default);
-    box-shadow: var(--component-button-primary-shadow);
+    box-shadow:
+      var(--shadow-size-sm)
+      var(--color-shadow-primary);;
```

Not a breaking issue in CSS 
```suggestion
      var(--color-shadow-primary);
```